.storybook/templates/DocumentationTemplate.mdx

```diff
@@ -1,5 +1,3 @@
-{/* DocumentationTemplate.mdx */}
```

Comment:
The build "test" command does _not_ like these comments! Easier to just remove them than try to rework them.